platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/PolicyManager.java

```diff
@@ -481,24 +442,10 @@ public void setRealms(List<String> realms) {
         }
     }
 
-    public void setRealms(String realms) {
-        LOGGER.debug("setRealms(String) called with {}", realms);
-        policyProperties.put(REALMS, realms);
-    }
-
     public void setWhiteListContexts(List<String> contexts) {
         LOGGER.debug("setWhiteListContexts(List<String>) called with {}", contexts);
         if (contexts != null && !contexts.isEmpty()) {
-            whiteListContexts = expandStrings(contexts);
-        }
-    }
-
-    public void setWhiteListContexts(String contexts) {
-        LOGGER.debug("setWhiteListContexts(String) called with {}", contexts);
-        if (StringUtils.isNotEmpty(contexts)) {
-            contexts = PropertyResolver.resolveProperties(contexts);
-            String[] contextsArr = contexts.split(",");
-            whiteListContexts = Arrays.asList(contextsArr);
+            whiteListContexts = contexts;
```

Comment:
This will not work with the new variable replacement that was added. WhiteListContexts strings can contain variables that must be replaced on ingest. Looks like with the way it is written right now we need to loop through the list and replace each value with output from  running PropertyResolver.resolveProperties on each item.